### DISCUSSION TITLE
NFS and/or SMB client access?

### DISCUSSION THREAD

**Discussion by alexn42** [seaweedfs/seaweedfs#2805] [General]
I don't see mention anywhere of being able to access the repositories via standard NFS or SMB clients?  Are there traditional "legacy" methods for users who need to put/get files in the old-style of mounting a drive letter or a mountpoint to a server?

**Comment by kmlebedev** [comment]
https://github.com/wthorp/GoSMB
Smb server not available for golang

**Reply by alexn42** [reply]
Looks like using "weed mount ..." (to mount a FUSE instance on a Linux host) and then exporting using NFS or SMB would be option?

**Reply by Stitch10925** [reply]
What happens if you run weed mount on windows, does it mount as harddrive? The documentation wasn't clear on that. If it does, that might bea way of accessing the data either directly or by sharing the mount.

**Reply by chrislusf** [reply]
'weed mount" is not available on windows as of now.

**Comment by kmlebedev** [comment]
@chrislusf Hi, how about adding NFS support?
Looks like there is a library https://github.com/willscott/go-nfs

https://confluence.atlassian.com/enterprise/example-data-center-configurations-935363786.html - what is the profit
Atlassian use NFS shared Storage

https://github.com/seaweedfs/seaweedfs/issues/3972

**Reply by chrislusf** [reply]
need a good library. go-nfs does not seem well tested.

**Reply by chrislusf** [reply]
added NFSv3 in https://github.com/seaweedfs/seaweedfs/pull/9067
